sections/interfaces/rest.md

```diff
@@ -290,3 +290,50 @@ If the gateway does not receive a HTTP 200 acknowledge it will try resending the
 
 **IMPORTANT:** Respond as fast as possible on the request before you start doing heavy business logic. It is strongly recommended to use an asynchronous model where you acknowledge the message and add it to an internal queue, for example database. Do not use a synchronous model where you do all the business logic and then send a response message. Our experience is that customers that fail to respond within the time limit of the request often experience that their messages are queued. This is because the system will use some of its resources to resubmit messages that your server has received earlier but failed to acknowledge. The higher the traffic peaks, the more significant this problem will be.
 
+## Authentication and authorization for receiving incoming SMS (MO) and Delivery Report (DR) messages
+
+Puzzel SMS GW supports the following methods for authentication and authorization of requests to deliver incoming SMS and delivery report messages:
+
+1. IP filtering
+2. Query parameter
+3. Header with a specific value
+4. OAuth2 Bearer tokens 
+
+Notice that for (3) and (4) the authentication configuration is used for all MO and DR messages for a service. It cannot be configured for each endpoint.
+
+### IP filtering
+IP filtering can be used to filter on the IP address of the client that sends the request. See [above](#Receiving-messages-(MO)) for the IP address.
+
+### Query parameter
+
+A query parameter may be added to the MO/DR endpoint. This must be a fixed value.
+
+### Header with a specific value
+
+Adds a header to requests with a given name and value.
+
+A few examples how this method can be used:
+Header name | Header value | Description
+----------- | -------------| -------------------
+X-API-Key   | 3141592654   | For a fixed API-key
+Authenticaiton | Bearer <jwt> | For a fixed bearer token, where <jwt> is a [JSON Web Token (JWT)](https://tools.ietf.org/html/rfc7519).
```

Comment:
Authentication, small typo

Reply:
Changed to Authorization, ref your other comment.